### PR TITLE
add import and export in index

### PR DESCRIPTION
**Pull request by TheRealAstoo**
## Description

I'm adding the import and export of the function `generateJWE` in the index.

## Motivation and Context

I could not use it in Sparta-API, so I came here 😄 

## How Has This Been Tested?

I did not test it.

## Types of changes

- [x] Chore (non-breaking change which refactors / improves the existing code base)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
  change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] My change requires a change to a package version.
- [x] I have updated the `package.json` version accordingly.
- [x] I have updated the `CHANGELOG.md` version accordingly.
- [x] I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

[CONTRIBUTING_FILE]: https://github.com/fewlinesco/node-web-libraries/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by Fenntasy** [comment]
Task linked: [CU-jb6590 [chore] - add generateJWE export](https://app.clickup.com/t/jb6590)

**Comment by TheRealAstoo** [comment]
Big changes here 😎

**Comment by Fuzznimp** [comment]
You need to bump the version and add the entry to the changelog.

**Comment by Oluane** [comment]
You need change the version in `package.json`, add a line in the `CHANGELOG` about your changes and then publish a new version of the package 🙂

**Comment by TheRealAstoo** [comment]
Thanks for your answers guys 😃

**Comment by TheRealAstoo** [comment]
Is it ok for the changelog ? It was my first 😄

**Comment by TheRealAstoo** [comment]
Do I need to do something else to push it on NPM or it's automatic / done by one of you only?

**Comment by Fuzznimp** [comment]
> Is it ok for the changelog ? It was my first 😄

No worries, we'll comment if needed.

> Do I need to do something else to push it on NPM or it's automatic / done by one of you only?

You need to:
- checkout to main
- pull main
- `npm longin` (credentials are in the team vault)
- `npm publish --access public` (from the package root folder)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Related Issue
+
+<!--- This project only accepts pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an -->
+<!--- issue first -->
+<!--- If fixing a bug, there should be an issue describing it with -->
+<!--- steps to reproduce -->
+<!--- Please link to the issue here with the prefix or resolve -->
+<!--- keyword -->
```
Comment (Fuzznimp) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: This is not needed, as the bot link it through the task ID from the branch's name.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests -->
+<!--- you ran to see how your change affects other areas of the -->
+<!--- code, etc. -->
+
+## Screenshots
+
+<!--- if appropriate, otherwise the section can be removed -->
+
```
Comment (Fuzznimp) [inline .github/PULL_REQUEST_TEMPLATE.md:31]: This is not needed as there is no frontend.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+<!--- Go over all the following points, and replace the `:red_circle:` in all -->
+<!--- the lines with a :white_check_mark: when relevant. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're -->
+<!--- here to help! -->
+- :red_circle: My code follows the code style of this project.
+- :red_circle: My change requires a change to the documentation.
+- :red_circle: I have updated the documentation accordingly.
+- :red_circle: I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
+- :red_circle: I have added tests to cover my changes.
+- :red_circle: All new and existing tests passed.
```
Comment (Fuzznimp) [inline .github/PULL_REQUEST_TEMPLATE.md:53]: You should use this checklist https://github.com/fewlinesco/node-web-libraries/blob/main/.github/PULL_REQUEST_TEMPLATE.md/

File: client/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.4.5] - 2021-05-05
+
+- Added the export of `generateJWE`
+
```
Comment (Fuzznimp) [inline client/CHANGELOG.md:8]:
```suggestion

- Added the export of `generateJWE`.

```

File: client/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.4.5] - 2021-05-05
+
```
Comment (Fuzznimp) [inline client/CHANGELOG.md:6]:
```suggestion

## [0.4.6] - 2021-05-05

```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (TheRealAstoo) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: Can be cool for reviewers to have a link?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (TheRealAstoo) [reply .github/PULL_REQUEST_TEMPLATE.md:53]: I prefer it to. Faster to look at or change something after.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Fuzznimp) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: The link will be added automatically by the bot
image

Comment (TheRealAstoo) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: True. I change it 👍

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (Meyclem) [reply .github/PULL_REQUEST_TEMPLATE.md:53]: image
😏

Comment (TheRealAstoo) [reply .github/PULL_REQUEST_TEMPLATE.md:53]: Hum, my bad. I took the Sparta version (that we should maybe change then ?)

Comment (Fuzznimp) [reply .github/PULL_REQUEST_TEMPLATE.md:53]: `sparta-monorepo` and `connect-js` are not the same kind, so I'd say no. I mean, you'd still have to provide screenshots, but we don't do versionning on `sparta-monorepo`.

Comment (TheRealAstoo) [reply .github/PULL_REQUEST_TEMPLATE.md:53]: Oh, I was talking about the circles vs the checks 👍

File: client/CHANGELOG.md [hunk 6]
```diff
@@ -2,9 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.4.5] - 2021-05-05
+## [0.4.6] - 2021-05-05
 
-- Added the export of `generateJWE`
+- Added the export of `generateJWE`.
```
Comment (Meyclem) [inline client/CHANGELOG.md:7]:
```suggestion
- Added the export of `generateJWE`. This was missing from `0.4.4`
```
Otherwise it could raise questions. This way we give all the need information about why this change is needed.

**Review by Meyclem — CHANGES_REQUESTED** [review]

**Comment by Meyclem** [comment]
And you can also update the PR description 🙂

**Comment by TheRealAstoo** [comment]
> description

With the new template?

File: client/CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.4.6] - 2021-05-05
+
+- Added the export of `generateJWE`. This was missing from `0.4.4`
```
Comment (Meyclem) [inline client/CHANGELOG.md:7]:
```suggestion
- Added the export of `generateJWE`. This was missing in the `0.4.4` version.
```
You were too quick to make the change ;)

Comment (TheRealAstoo) [reply client/CHANGELOG.md:7]: 😄

**Comment by Meyclem** [comment]
> > And you can also update the PR description 🙂
> 
> With the new template?

Yes

**Comment by TheRealAstoo** [comment]
> > > And you can also update the PR description 🙂
> > 
> > 
> > With the new template?
> 
> Yes

Just seen something weird in the template do we put circles or cross? 🤔 : 





- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to a package version.
- [ ] I have updated the `package.json` version accordingly.
- [ ] I have updated the `CHANGELOG.md` version accordingly.
- [ ] I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

**Comment by Fuzznimp** [comment]
> Just seen something weird in the template do we put circles or x check? 🤔 

Just put x.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] My change requires a change to a package version.
+- [ ] I have updated the `package.json` version accordingly.
+- [ ] I have updated the `CHANGELOG.md` version accordingly.
+- [ ] I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
+- [ ] I have added tests to cover my changes.
+- [ ] All new and existing tests passed.
+
+[CONTRIBUTING_FILE]: https://github.com/fewlinesco/node-web-libraries/blob/master/CONTRIBUTING.md
```
Comment (Fuzznimp) [inline .github/PULL_REQUEST_TEMPLATE.md:41]: This points to an unrelated repo. I'd say we remove it for now, until we have a proper file for the monorepo.

**Review by Meyclem — APPROVED** [review]
